### PR TITLE
Removing early note

### PR DESCRIPTION
**Pull request by get-itips**
@SerdarSoysal please kindly check.
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6336

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/communicate-with-users-from-other-organizations.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ At a high level, here’s how to choose (for a detailed comparison, jump down to
 
 ## External access
 
-Use **external access** (federation) when you need a solution that lets external users in other domains find, call, chat, and set up meetings with you. External users have no access to your organization's teams or team resources. Choose external access when you want to communicate with users outside your organization who are still on Skype for Business (online or on premises) or Skype (coming in early 2020). 
+Use **external access** (federation) when you need a solution that lets external users in other domains find, call, chat, and set up meetings with you. External users have no access to your organization's teams or team resources. Choose external access when you want to communicate with users outside your organization who are still on Skype for Business (online or on premises) or Skype.
```
Comment (JohanFreelancer9) [inline Teams/communicate-with-users-from-other-organizations.md:33]:
```suggestion
Use **external access** (federation) when you need a solution that lets external users in other domains find, call, chat, and set up meetings with you. External users have no access to your organization's teams or team resources. Choose external access when you want to communicate with users outside your organization who are still on Skype for Business (online or on-premises) or Skype. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor additional edit, please.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
Can we merge this one ? :)

**Comment by get-itips** [comment]
@SerdarSoysal Can you kindly review and merge? Thank you!

**Comment by SerdarSoysal** [comment]
Closing this PR without merging because the same update was made via a different PR and the live article is already updated as a result.

**Pull request closed without merging**